src/utils.ts

```diff
@@ -51,8 +93,9 @@ export const createServer = () => {
 
   // Base route
   app.use("/", (_req, res) => {
-    res.status(404).send("Endpoint does not exist.");
+    res.status(404).send("Endpoint does not exist");
```

Comment:
lol

Reply:
I was testing something by changing the text for this. Didn't mean to remove the period lmao